Makefile

```diff
@@ -1,7 +1,13 @@
 .PHONY: test
 
+# Disable CGO by default.
+export CGO_ENABLED=0
+
 bin/ssm-env: *.go
-	CGO_ENABLED=0 go build -o $@ .
+	# Enforce static linking, and copy the timezone data into the runtime so we
+	# can in places like scratch images.
+	go build -ldflags '-extldflags "-static"' -tags timetzdata -o $@ .
```

Comment:
Does ssm-env actually need tzdata for something?

Reply:
Not that I know of, but I figured it'd be safer to include it rather than have a very bizarre error far away from this repo (if someone modifies ssm-env to need it)

Reply:
Perhaps, though normally I'd expect the onus to be on the installer to ensure they have an up-to-date timezone database available in the environment. For republic I think we should probably include tzdata in the image environment anyway (like ca-certificates) so it's the same for republic itself and ssm-env, and can be updated independently.